L1TriggerConfig/L1GtConfigProducers/python/l1GtTriggerMenuXml_cfi.py

```diff
@@ -14,4 +14,17 @@
     VmeXmlFile = cms.string('')
 )
 
+#
+# Make changes for Run 2
+#
+from Configuration.StandardSequences.Eras import eras
+# Change the trigger menu depending on whether this is 25ns, 50ns or HI running
+eras.run2_25ns_specific.toModify( l1GtTriggerMenuXml, DefXmlFile = 'L1Menu_Collisions2015_25ns_v2_L1T_Scales_20141121_Imp0_0x1030.xml' )
 
+## The following two menus haven't been implemented yet, so until then use
+## the same menu as for 25ns.
+#eras.run2_50ns_specific.toModify( l1GtTriggerMenuXml, DefXmlFile = 'L1Menu_Collisions2015_50ns_v0_L1T_Scales_20141121_Imp0_0x1031.xml' )
+#eras.run2_HI_specific.toModify( l1GtTriggerMenuXml,   DefXmlFile = 'L1Menu_CollisionsHeavyIons2011_v0_nobsc_notau_centrality_q2_singletrack.v1.xml' )
+## FIXME - take out these lines and uncomment the ones above once the menus are implemented.
```

Comment:
@fwyzard These lines will need to be changed for different menus for 25ns, 50ns or heavy ions.  You can add variables to make it easier if you want e.g.

    L1Menu_Collisions2015_50ns_v0 = 'L1Menu_Collisions2015_50ns_v0_L1T_Scales_20141121_Imp0_0x1031.xml'
    L1Menu_Collisions2015_50ns_v1 = 'something'
    eras.run2_50ns_specific.toModify( l1GtTriggerMenuXml, DefXmlFile = L1Menu_Collisions2015_50ns_v0 )

I'll leave that up to you.